File: amason/frontend/src/vista-pedidosenviados.js
import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
import '@vaadin/vaadin-ordered-layout/src/vaadin-vertical-layout.js';

class VistaPedidosenviados extends PolymerElement {

    static get template() {
        return html`
<style include="shared-styles">
                :host {
                    display: block;
                    height: 100%;
                }
            </style>
<vaadin-vertical-layout id="layout-pedidos-enviados"></vaadin-vertical-layout>
`;
    }

    static get is() {
        return 'vista-pedidosenviados';
    }

    static get properties() {
        return {
            // Declare your properties here.
        };
    }
}

customElements.define(VistaPedidosenviados.is, VistaPedidosenviados);

File: amason/src/main/java/vistas/VistaPedidosenviados.java
package vistas;

import com.vaadin.flow.component.Tag;
import com.vaadin.flow.component.dependency.JsModule;
import com.vaadin.flow.component.orderedlayout.VerticalLayout;
import com.vaadin.flow.component.polymertemplate.PolymerTemplate;
import com.vaadin.flow.templatemodel.TemplateModel;
import com.vaadin.flow.dom.Element;
import com.vaadin.flow.component.polymertemplate.Id;

/**
 * A Designer generated component for the vista-pedidosenviados template.
 *
 * Designer will add and remove fields with @Id mappings but
 * does not overwrite or otherwise change this file.
 */
@Tag("vista-pedidosenviados")
@JsModule("./src/vista-pedidosenviados.js")
public class VistaPedidosenviados extends PolymerTemplate<VistaPedidosenviados.VistaPedidosenviadosModel> {

    @Id("layout-pedidos-enviados")
    private VerticalLayout layoutPedidosEnviados;

    public VerticalLayout getLayoutPedidosEnviados() {
        return layoutPedidosEnviados;
    }

    public void setLayoutPedidosEnviados(VerticalLayout layoutPedidosEnviados) {
        this.layoutPedidosEnviados = layoutPedidosEnviados;
    }

    /**
     * Creates a new VistaPedidosenviados.
     */
    public VistaPedidosenviados() {
        // You can initialise any data required for the connected UI components here.
    }

    /**
     * This model binds properties between VistaPedidosenviados and vista-pedidosenviados
     */
    public interface VistaPedidosenviadosModel extends TemplateModel {
        // Add setters and getters for template properties here.
    }
}
